website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -63,15 +63,15 @@ Properties without a default value are required.
 | `data.attributes.auto-apply`                  | boolean | `false`   | Whether to automatically apply changes when a Terraform plan is successful, [with some exceptions](/cloud-docs/workspaces/settings#auto-apply-and-manual-apply).                                                                                                                                                                                                                                                                                                                                                |
 | `data.attributes.description`                 | string  | (nothing) | A description for the workspace.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
 | `data.attributes.execution-mode`              | string  | `remote`  | Which [execution mode](/cloud-docs/workspaces/settings#execution-mode) to use. Valid values are `remote`, `local`, and `agent`. When set to `local`, the workspace will be used for state storage only. This value must not be specified if `operations` is specified.                                                                                                                                                                                                                                          |
-| `data.attributes.file-triggers-enabled`       | boolean | `true`    | Whether to filter runs based on the changed files in a VCS push. If enabled, the `working-directory` and `trigger-prefixes` describe a set of paths which must contain changes for a VCS push to trigger a run. If disabled, any push will trigger a run.                                                                                                                                                                                                                                                       |
+| `data.attributes.file-triggers-enabled`       | boolean | `true`    | Whether to filter runs based on the changed files in a VCS push. If enabled it uses either `trigger-prefixes` in conjunction with `working_directory` or `trigger-patterns` to describe a set of triggers upon which a run will be triggered. If disabled, any push will trigger a run.                                                                                                                                                                                                                         |
```

Comment:
A couple nits here and then I think we should (once those docs are written) link to the section that will contain info about prefixes and patterns.

```suggestion
| `data.attributes.file-triggers-enabled`       | boolean | `true`    | Whether to filter runs based on the changed files in a VCS push. If enabled, it uses either `trigger-prefixes` in conjunction with `working_directory` or `trigger-patterns` to describe a set of events that will start a run. If disabled, any push will trigger a run. Refer to [Automatic Run Triggering](/cloud-docs/workspaces/settings/vcs#automatic-run-triggering) for details.                                                                                                                                                                                                                         |
```

Reply:
IMHO `events` sounds a bit generic, especially because we'll have additional trigger types (the one based on git tags). These two types (`trigger-prefixes` and `trigger-patterns`) are concerned with the changed files and their paths. Could there be a better way of emphasizing that? Maybe `to describe paths of changed files that will start a run`?

Reply:
this is a very fair point - good call. I like "to describe the set of changed files that will start a run" - do it!